### PR TITLE
feature: enable to use CRDs imported as modules

### PR DESCRIPTION
**Pull request by 9bany**
### Proposed changes
- issue link: https://github.com/nginxinc/kubernetes-ingress/issues/3770
- description: able to import module with `/v3` 

### Checklist
Before creating a PR, run through this checklist and mark each as complete.

- [x] I have read the [CONTRIBUTING](https://github.com/nginxinc/kubernetes-ingress/blob/main/CONTRIBUTING.md) doc
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked that all unit tests pass after adding my changes
- [x] I have updated necessary documentation
- [x] I have rebased my branch onto main
- [x] I will ensure my PR is targeting the main branch and pulling from my branch from my own fork

### CODE REVIEW

**Comment by jjngx** [comment]
@9bany can I ask you to update your remote and push changes please?

**Comment by jjngx** [comment]
@9bany could you please update your branch, run `make update-codegen` and then push?

**Comment by vepatel** [comment]
ping @9bany

**Comment by 9bany** [comment]
let me pick it up !

**Review by jjngx — APPROVED** [review]
👍🏻

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
 # Changelog
+## Changed
```
Comment (haywoodsh) [inline CHANGELOG.md:2]: The change logs should not be edited as it is automatically generated

**Review by shaun-nx — APPROVED** [review]

File: internal/certmanager/test_files/context_builder.go [hunk 2]
```diff
@@ -38,9 +38,10 @@ import (
 	cmfake "github.com/cert-manager/cert-manager/pkg/client/clientset/versioned/fake"
 	informers "github.com/cert-manager/cert-manager/pkg/client/informers/externalversions"
 	"github.com/cert-manager/cert-manager/pkg/util"
-	k8s_nginx "github.com/nginxinc/kubernetes-ingress/pkg/client/clientset/versioned"
-	vsfake "github.com/nginxinc/kubernetes-ingress/pkg/client/clientset/versioned/fake"
-	vsinformers "github.com/nginxinc/kubernetes-ingress/pkg/client/informers/externalversions"
+
```
Comment (haywoodsh) [inline internal/certmanager/test_files/context_builder.go:41]: This empty line looks redundant.

**Review by haywoodsh — APPROVED** [review]

**Pull request merged**